src/main/scala/ai/privado/exporter/SinkExporter.scala

```diff
@@ -171,7 +178,16 @@ class SinkExporter(cpg: Cpg, ruleCache: RuleCache) {
         .filter(node => node.name.equals(Constants.id) || node.name.startsWith(Constants.privadoDerived))
       node.value.toSet
     }
-    sinks
+
+    val updatedSinks = sinks.map { tags =>
+      val containsAPIUrl = tags.exists(isAPIUrlTag)
+      if (containsAPIUrl) {
+        tags.filterNot(tag => tag.value == Constants.thirdPartiesAPIRuleId)
+      } else {
+        tags
+      }
+    }
+    updatedSinks
```

Comment:
Have you tested this, doesn't look like this will work

Reply:
removed will work again with different approach.